modules/authentication/src/handlers/local.ts

```diff
@@ -161,18 +161,20 @@ export class LocalHandlers implements IAuthenticationStrategy {
       this.verifyChangeEmail.bind(this),
     );
 
-    routingManager.route(
-      {
-        path: '/local/resend-verification',
-        action: ConduitRouteActions.POST,
-        description: `Used to resend email verification after new user is created`,
-        bodyParams: {
-          email: ConduitString.Required,
+    if (this.sendVerificationEmail) {
```

Comment:
This should only register whenever `this.sendVerificationEmail` is true.
Move this into the relevant block above.